src/odemis/driver/xt_client.py

```diff
@@ -2147,10 +2166,17 @@ class Stage(model.Actuator):
     moving the TFS stage and updating the position.
     """
 
-    def __init__(self, name: str, role: str, parent: model.HwComponent, rng: dict = None, **kwargs) -> None:
+    def __init__(self, name: str, role: str, parent: model.HwComponent, rng: dict = None, raw: bool = False,
+                 **kwargs) -> None:
+        self.raw = raw
+        self.parent = parent
+        # define axes which needs offset correction such that TFS stage coordinates
+        # are equal to stage coordinates displayed through Odemis
```

Comment:
Please explain more this comment. At least you should explain that 1) it's related to the raw coordinates, and 2) it'll be updated later in the init.